include/rosidl_dynamic_typesupport/uchar.h

```diff
@@ -22,10 +22,12 @@ extern "C" {
 #if defined(__cplusplus) && __cplusplus >= 201103L
 // Nothing to do here, C++11 and beyond have char16_t as a keyword:
 // https://en.cppreference.com/w/cpp/keyword/char16_t
-#elif defined(__has_include) && __has_include(<uchar.h>)
+#elif defined(__has_include)
+#  if __has_include(<uchar.h>)
```

Comment:
Wasn't sure about the formatting here, but the `include <uchar.h>` below was/is indented before I changed it, so I tried to use the same style.